Automattic/wp-calypso
# Proof of Concept: Use isolated block editor for reader comments

#### Changes proposed in this Pull Request

* Copies `isolated-editor` from the private Automattic/p2tenberg repository
* Uses it for reader comments

#### Testing instructions



* Post a comment in reader using the block editor. Verify that you are able to save the comment and have it be posted.
